### PR TITLE
Fix flakiness in fixed batch size integration test

### LINKED ISSUE #386: Fixed batch size integration test is flaky
`test_fixed_batch_size_model_processes_specified_query_batch_size_when_saturated` from the Clipper manager integration tests is flaky. After discussing with @Corey-Zumar, sticking a `time.sleep` in the model container to allow Clipper enough time to enqueue all the requests should address the issue.

### PR DESCRIPTION
**Pull request by Corey-Zumar**
Fixes #386

### CODE REVIEW

**Review by dcrankshaw — APPROVED** [review]
lgtm

**Comment by dcrankshaw** [comment]
Hmm @Corey-Zumar looks like this maybe doesn't fix the flakiness https://amplab.cs.berkeley.edu/jenkins/job/Clipper-PRB/936/console

**Comment by Corey-Zumar** [comment]
@dcrankshaw Based on testing with AWS, it appears that the container crashes when queried. No stack trace is produced. The issue appears to be associated with the serialization and deserialization of functions that call `time.sleep` or `Timer.Start()`.

For now, in place of `time.sleep()`, it seems reasonable to perform a higher-latency computation before returning the batch sizes as outputs. Do you agree?

**Comment by dcrankshaw** [comment]
Hmm weird. You could just hardcode a new testing docker container that has a sleep in it (similar to the noop container). That seems like a potentially cleaner and more controlled solution.

**Comment by Corey-Zumar** [comment]
Good call. Will do that.

**Comment by Corey-Zumar** [comment]
@dcrankshaw The container-based approach doesn't fare any better. I'm still encountering the same traceless crash. Will need to debug.

File: dockerfiles/PyClosureContainerDockerfile [hunk 1]
```diff
@@ -3,12 +3,11 @@ FROM clipper/py-rpc:${CODE_VERSION}
...
 RUN conda config --set ssl_verify no \
+  && conda install -y -c anaconda cloudpickle=0.5.2 \
   && conda install -y --file /lib/python_container_conda_deps.txt
-RUN conda install -c anaconda cloudpickle=0.5.2
 
 COPY containers/python/python_closure_container.py containers/python/python_closure_container_entry.sh /container/
 
-
-CMD ["/container/python_closure_container_entry.sh"]
+ENTRYPOINT ["/container/python_closure_container_entry.sh"]
```
Comment (Corey-Zumar) [inline dockerfiles/PyClosureContainerDockerfile:11]: Changing this from `CMD` to `ENTRYPOINT` fixes the crash associated with the python call to `time.sleep`. I'll have to explore why this is the case. `CMD` executes the python script within `/bin/sh -c` directive, while `ENTRYPOINT` executes the python script in some other fashion; I'd imagine that this accounts for the difference.

File: dockerfiles/PyClosureContainerDockerfile [hunk 2]
```diff
@@ -3,12 +3,11 @@ FROM clipper/py-rpc:${CODE_VERSION}
 
 COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
 RUN conda config --set ssl_verify no \
+  && conda install -y -c anaconda cloudpickle=0.5.2 \
```
Comment (Corey-Zumar) [inline dockerfiles/PyClosureContainerDockerfile:6]: Installing this first fixes an installation error.

**Pull request merged**